index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>To Do List</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="./css/styles.css" />
+</head>
+<body>
+    <header>
+           <h2>
+               To Do List
+           </h2>                                    
+    </header>
+    <section>
+        <form id="item-form">
+            <label>
+                Item To Do:
+                <input type="text" required name="todo">
+            </label>
+            <label>
+                Due Date:
+                <input type="date" required name="due">
+            </label>
+            <button type="submit" name="submit">Submit</button>
+        </form>
+    </section>
+    <section>
+        <h2>To Do:</h2>
+        <ul id="listItems">listItems</ul>
+    </section>
+    <script type="module" src="/js/app.js"></script>
```

Comment:
```suggestion
    <script type="module" src="js/app.js"></script>
```
GitHub pages is sensitive to the relative file src having an extra ```/``` in it.